languages/java/exercises/concept/conditionals/.docs/hints.md

```diff
@@ -0,0 +1,45 @@
+## General
+
+Conditionals are used to check for certain conditions and/or criteria. The most basic way of performing a conditional operation is using a single `if` statement.
+
+## 1. Calculate the score of any given card.
+
+The `ParseCard` function should take the `card` string (e.g. `ace`) and turn it into its value (e.g. 11).
```

Comment:
functions names in java normally hasn't capital letter, so prefer `parseCard` instead.
please fix this in all the markdown file